src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -0,0 +1,28 @@
+__PROJECT_DIR=$1
+__TOOLS_DIR=$2
+__DOTNET_CMD=$3
+__TOOLRUNTIME_DIR=$__PROJECT_DIR/Tools
+
+__BUILDERRORLEVEL=0
+
+if [ ! -d "$__PROJECT_DIR" ]; then
+   echo "ERROR: Cannot find project root path at '$__PROJECT_DIR'. Please pass in the source directory as the 1st parameter."
+   exit 1
+fi
+
+if [ ! -d "$__TOOLS_DIR" ]; then
+   echo "ERROR: Cannot find tools path at '$__TOOLS_DIR'. Please pass in the tools directory as the 2nd parameter."
+   exit 1
+fi
+
+if [ ! -e "$__DOTNET_CMD" ]; then
+   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 3rd parameter."
+   exit 1
+fi
+
+$__DOTNET_CMD restore $__TOOLS_DIR/tool-runtime/project.json
+$__DOTNET_CMD publish $__TOOLS_DIR/tool-runtime/project.json -f dnxcore50 -r ubuntu.14.04-x64 -o $__TOOLRUNTIME_DIR
```

Comment:
I also do not have a mac to test on :smile:

That said:
`-r ubuntu.14.04-x64`
This will definitely cause some issues, especially for native components. 

A basic way to solve this is to do something like this:
```bash
if [[ "$(uname)" == "Darwin" ]]; then
    $RID="osx.10.10-x64"
elif [[ "$(uname)" == "Linux" ]]; then
    $RID="Ubuntu.14.04-x64"
fi
```

This doesn't solve the issue completely though, because there is no concept of version or what flavor of Linux you are on.